Automattic/wp-calypso
# React-Query: Fix Happychat persisting config

Related to [this comment](https://github.com/Automattic/wp-calypso/pull/66407#discussion_r1173430179)

## Proposed Changes

* This uses proper config to prevent React query caching. 
* It removes the ever-changing key as a cache buster and instead disables persisting. 
* Having the key there meant spamming the IndexDB cache because we did cache, but under a different key everytime.

## Testing Instructions
1. Login to https://hud-staging.happychat.io/. Don't set yourself available.
2. Go to the live link below.
3. Open the Help Center, click "Still need help", chat should not be available. 
4. Make yourself available in HUD.
5. Refresh the Calypso page.
6. Open the Help Center.
7. Chat should be available.
